jina/serve/runtimes/request_handlers/data_request_handler.py

```diff
@@ -18,7 +18,7 @@
     from jina.logging.logger import JinaLogger
 
 
-class DataRequestHandler:
+class ExecutorRequestHandler:
```

Comment:
I think `WorkerRequestHandler` would be even better, since internally we have the `WorkerRuntime` that handles Executors, and also uses this handler.